Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lights/Ltc.azsli

```diff
@@ -445,29 +506,32 @@ void EvaluatePolyEdge(in float3 p0, in float3 p1, inout float3 prevClipPoint, in
 // EvaluatePolyEdge() later. During this search it also adjusts the end point index as necessary to avoid processing
 // those points that are below the horizon.
 void LtcPolygonEvaluate(
-    in float3 pos, 
-    in float3 normal, 
-    in float3 dirToView,
-    in float3x3 ltcMat,
+    in Surface surface,
+    in LightingData lightingData,
+    in Texture2D ltcMatrix,
+    //in Texture2D<float2> ltcAmpMatrix,
```

Comment:
Any reason for the comment? I see other places where this is called with the comment as well.

Reply:
Was debugging an issue and forgot to put this back. Will fix.